File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        
        int value1 = int.Parse(a.ToString() + b.ToString());
        int value2 = 2 * a * b;
        
        if (value1 >= value2) {
            answer = value1;
        }
        else {
            answer = value2;
        }
        return answer;
    }
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) 
{
    string sum_ab = to_string(a) + to_string(b);
    int answer = 0;
    if(stoi(sum_ab) >= 2 * a * b)
    {
        answer = stoi(sum_ab);
    }
    else
    {
        answer = 2 * a * b;
    }
    return answer;
}